roachtest: psycopg failed

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4962211&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4962211&tab=artifacts#/psycopg) on release-22.1 @ [fd435e6d411a79425eba7ecdf82f0efb66025550](https://github.com/cockroachdb/cockroach/commits/fd435e6d411a79425eba7ecdf82f0efb66025550):


```
The test failed on branch=release-22.1, cloud=gce:
test artifacts and logs in: /artifacts/psycopg/run_1
	orm_helpers.go:190,orm_helpers.go:116,psycopg.go:144,psycopg.go:156,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-beta.3-137-gfd435e6d41
		Tests run against psycopg 2_8_6
		453 Total Tests Run
		451 tests passed
		2 tests failed
		319 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_connection.TestEncryptPassword.test_encrypt_server (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blocklist (psycopgBlockList22_1) is available in the artifacts' psycopg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/orm_helpers.go
+++ b/pkg/cmd/roachtest/tests/orm_helpers.go
@@ -49,6 +49,9 @@ func alterZoneConfigAndClusterSettings(
 		`SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;`,
 		`SET CLUSTER SETTING kv.range_split.by_load_merge_delay = '5s';`,
 
+		// Test with SCRAM password authentication.
+		`SET CLUSTER SETTING server.user_login.password_encryption = 'scram-sha-256';`,
+
 		// Enable experimental features.
 		`SET CLUSTER SETTING sql.defaults.experimental_temporary_tables.enabled = 'true';`,
 		`SET CLUSTER SETTING sql.defaults.datestyle.enabled = true`,

